src/libultra/gu/coss.c

```diff
@@ -1,4 +1,4 @@
-#include "global.h"
+#include "guint.h"
```

Comment:
```suggestion
#include "gu.h"
```
Note I don't know what you're going for with includes

I assume they should be minimal?
But for example would a
```c
#include "guint.h"
```
->
```c
#include "ultratypes.h"
#include "gu.h"
```
be better if something didn't need what guint.h itself contains?

Reply:
I've mostly been using the `*int` headers for each libultra module, since I believe that's how it was originally based on mdebug includes, but maybe a more minimal approach is just better idk. Though note that as I mention below the minimal set of includes must also contain the prototype for the functions in the file, not just the necessary external declarations that the function(s) uses.